nsdb-core/src/main/scala/io/radicalbit/nsdb/index/FacetSumIndex.scala

```diff
@@ -42,10 +42,13 @@ class FacetSumIndex(override val directory: Directory, override val taxoDirector
   override def write(bit: Bit)(implicit writer: IndexWriter, taxonomyWriter: DirectoryTaxonomyWriter): Try[Long] = {
 
     def facetWrite(f: Field, path: String, c: FacetsConfig): Field = bit.value match {
-      case x: java.lang.Integer => new IntAssociationFacetField(x, f.name, path)
-      case x: java.lang.Long    => new LongAssociationFacetField(x, f.name, path)
-      case x: java.lang.Float   => new FloatAssociationFacetField(x, f.name, path)
-      case x: java.lang.Double  => new DoubleAssociationFacetField(x, f.name, path)
+      case NSDbIntType(rawValue)    => new IntAssociationFacetField(rawValue, f.name, path)
+      case NSDbLongType(rawValue)   => new LongAssociationFacetField(rawValue, f.name, path)
+      case NSDbDoubleType(rawValue) => new DoubleAssociationFacetField(rawValue, f.name, path)
+//      case x: java.lang.Integer => new IntAssociationFacetField(x, f.name, path)
+//      case x: java.lang.Long    => new LongAssociationFacetField(x, f.name, path)
+//      case x: java.lang.Float   => new FloatAssociationFacetField(x, f.name, path)
+//      case x: java.lang.Double  => new DoubleAssociationFacetField(x, f.name, path)
```

Comment:
Are those comments wanted?

Reply:
Absolutely not :)  Thanks for reporting it